content/applications/general/apps_modules.rst

```diff
@@ -2,73 +2,84 @@
-   subscription costs.
-
-   - | **Installing or uninstalling apps and managing users is up to you.**
-     | As the administrator of your database, you are responsible for its usage, as you know best
-       how your organization works.
-   - | **Odoo apps have dependencies.**
-     | Installing some apps and features with dependencies may also install additional apps and
-       modules that are technically required, even if you won't actively use them.
-   - | **Test app installation/removal on a duplicate of your database.**
-     | This way, you can know what app dependencies may be required or what data may be erased.
+   Adding or removing apps can significantly affect other apps in the database and modify
+   subscription costs. Consider carefully or test the changes in a staging environment before
+   proceeding.
+
+   - **Administrators manage the database**: The administrator of the database is responsible for
+     its usage, as they know best how their organization works.
+   - **Odoo apps can have dependencies**: Installing some apps and features with dependencies may
+     also install additional apps and modules that are technically required, even if database users
+     do not actively use them.
+   - **Duplicate the database to test apps**: Testing on a duplicate database reveals what app
+     dependencies may be required or what database may be erased. Learn how to duplicate an
+     :doc:`Odoo Online database <../../administration/odoo_online>` or an :doc:`Odoo On-premise
+     database <../../administration/on_premise>`.
 
 .. _general/install:
 
 Install apps and modules
 ========================
 
-Go to :menuselection:`Apps`, and click on the *Install* button of the app you want to install.
+From the database home page, click on the :guilabel:`Apps` app, then click on the search bar to
+find the app to be installed or scroll to find it. From here, click :guilabel:`Activate` on the
+app's card.
 
 .. note::
-   If the module you are looking for is not listed, you can **update the app list**.
-
-   To do so, activate the :ref:`developer mode <developer-mode>`, then go to :menuselection:`Apps
-   --> Update Apps List` and click on *Update*.
+   If the app or module to be installed is not listed, update the app list by activating
+   :ref:`developer mode <developer-mode>`, and then click on the :guilabel:`Apps`. Then, in the top
+   navigation menu, click :menuselection:`Update Apps List`, and then click :guilabel:`Update`.
 
 .. _general/upgrade:
 
 Upgrade apps and modules
 ========================
 
-On some occasions, new improvements or app features are added to :doc:`supported versions of Odoo
-</administration/supported_versions>`. To be able to use them, you must **upgrade** your app.
+With each :doc:`new Odoo release </administration/supported_versions>` (e.g., 17.3, 18.0), new
+improvements or app features are added. Upgrade the app to use these new improvements and features.
```

Comment:
I dont think there is a need to state any specific versions here, as the linked _supported_versions_ doc provides that additional context
```suggestion
With each :doc:`new Odoo release </administration/supported_versions>`, new improvements or app
features are added. Upgrade the app to use these new improvements and features.
```